lib/generators/avo/resource_generator.rb

```diff
@@ -266,13 +280,21 @@ def generated_fields_template
           fields_string = ""
 
           fields.each do |field_name, field_options|
-            # if field_options are not available (likely a missing resource for an association), skip the field
             fields_string += "\n    # Could not generate a field for #{field_name}" and next unless field_options
 
             options = ""
-            field_options[:options].each { |k, v| options += ", #{k}: #{v}" } if field_options[:options].present?
+            if field_options[:options].present?
+              field_options[:options].each { |k, v| options += ", #{k}: #{v}" }
+            end
 
-            fields_string += "\n    #{field_string field_name, field_options[:field], options}"
+            # Add a comment for polymorphic fields
+            if field_options[:field] == "polymorphic"
+              fields_string += "\n    # Polymorphic association detected for :#{field_name}"
+              fields_string += "\n    # Uncomment and configure the possible types:"
+              fields_string += "\n    # field :#{field_name}, as: :polymorphic, types: [User, Admin]"
+            else
```

Comment:
There isn't a field type `polymorphic`

[Here](https://github.com/avo-hq/avo/blob/main/spec/dummy/app/avo/resources/comment.rb#L22) is an example of a `belongs_to` field for a polymorphic association.